CleverTapSDK/CleverTap.m

```diff
@@ -3749,6 +3764,36 @@ - (void)showInAppNotificationIfAny {
     }
 }
 
+- (void)suspendInAppNotifications {
+    if ([[self class] runningInsideAppExtension]) {
+        CleverTapLogDebug(self.config.logLevel, @"%@: suspendInAppNotifications is a no-op in an app extension.", self);
+        return;
+    }
+    if (!self.config.analyticsOnly) {
+        self.inAppRenderingStatus = CleverTapInAppSuspend;
```

Comment:
Add logging as `CleverTapLogDebug(self.config.logLevel, @"%@: Please Note - InApp Notifications will be suspended till resumeInAppNotifications() is not called again", self);`